discohorts/pipeline.py

```diff
@@ -14,21 +14,16 @@
 
 from __future__ import print_function
 
-from types import FunctionType
 from subprocess import check_call
 from os import environ, path
 import time
+from types import FunctionType
+
 
 class Pipeline(object):
-    def __init__(self, name, ocaml_path, name_cli_arg, other_cli_args,
-                 patient_subset_function, work_dir_function,
-                 batch_size, batch_wait_secs):
-        self.name = name
+    def __init__(self, ocaml_path, config, batch_size, batch_wait_secs):
+        self.config = config
         self.ocaml_path = ocaml_path
```

Comment:
nit-pick: should we call this `pipeline_script_path` instead? This confused me a bit at first since I thought this was the path to the `ocaml` binary :man_shrugging: